plugins/modules/nomad_token.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Pedro Nascimento <apecnascimento@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_token
+author: Pedro Nascimento(@apecnascimento)
+version_added: "1.0.0"
+short_description: Manage Nomad ACL tokens
+description:
+    - Create Bootstrap token
+    - Create ACL token.
+    - Update ACL token.
+    - Delete ACL token
```

Comment:
Every item in this list is a paragraph of the description. It should consist of proper sentences. How about:
```suggestion
    - This module allows to create Bootstrap tokens, create ACL tokens, update ACL tokens, and delete ACL tokens.
```